character-count.js

```diff
@@ -0,0 +1,17 @@
+function characterCount(arr) {
+
+    var obj = {}
+    for (var i = 0; i < arr.length; i++) {
+        var key = arr[i].length;
+        if (!obj[key]) {
+            obj[key] = 1
+        } else {
+            obj[key] += 1;
```

Comment:
Again here, you can try to use a ternary:
```javascript
obj[key] = (obj[key] ? 1 : 0) + 1
```

or 

```javascript
obj[key] = obj[key] ? 1 : (obj[key] + 1);
```

or (not using ternary)

```javascript
obj[key] = (obj[key] || 0)  + 1
```

or (using a new javascript feature which is better than the previous)

```javascript
obj[key] = (obj[key] ?? 0)  + 1
```